docs/proposals/2022-06-27_SEO_Filename_proposal.md

```diff
@@ -0,0 +1,44 @@
+# SEO Filename Proposal
+
+# Problem
+
+Image resizer links in Arc are pretty ugly. The only reference to an image filename is the Arc ID, like `ABCDEFGHIJKLMNOPQRSTUVWXYZ.jpg`. If clients want to improve the SEO scoring of their image links in Google, they need to be able to make that filename more realistic.
+
+# Proposal
+
+Like Canonical URLs do for gallery leaf pages, we want to introduce a string field SEO Filename. This field will be used by PageBuilder and others when generating Resizer V2 URLs by appending it before the asset key, like `/resizer/v2/this-is-an-seo-filename-ABCDEFGHIJKLMNOPQRSTUVWXYZ.jpg`.
+
+## SEO Filename
+
+SEO Filename must be formatted like a slug, where the only allowed characters are alphanumeric and dashes. It must be at or below 96 characters.
+
+# Example
+
+```
+{
+    "_id": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    "additional_properties": {
+        ...
+    },
+    "caption": "This is a caption.",
+    ...
+    "seo_filename": "this-is-an-seo-filename",
```

Comment:
Is the plan to add this to just the Image Content-Type?

Reply:
Yup!